content/using.html

```diff
@@ -66,25 +66,28 @@
 
   {% endcall %}
 
-  {% call subsection('Channels')%}
+  {% call subsection('Labels')%}
 
-  Each file within a package may be tagged in one or more channels. A file may also not be tagged in any channels.
-  One common use of channels is to divide a package's files into development, testing, and main channels called 
-  dev, test, and main. This allows package authors to upload files for development or testing purposes without
-  affecting non-development users.
-
-  At [https://**pypi**.anaconda.org/conda](https://pypi.anaconda.org/conda) a channel
-  dropdown lists all the available channels in this user namespace.
+  Each file within a package may be tagged with one or more labels, or not tagged at all. 
+  Labels are commonly used to divide a package's files into sections labeled development, 
+  testing (and/or other custom labels), and main or dev, test, main or other custom label. 
```

Comment:
This sentence is pretty confusing. I can't tell what it is trying to convey - is it telling me to use the names `dev`, `test`, and `main` for my labels? I think we could remove it without much harm.